src/Postman/CollectionWriter.php

```diff
@@ -56,8 +56,23 @@ public function getCollection()
                         return [
                             'name' => $route['title'] != '' ? $route['title'] : url($route['uri']),
                             'request' => [
-                                'url' => url($route['uri']),
+                                'url' => url($route['uri']).(collect($route['queryParameters'])->isEmpty()
+                                    ? ''
+                                    : ('?'.implode('&', collect($route['queryParameters'])->map(function ($parameter, $key) {
+                                        return $key.'='.(isset($parameter['value']) ? $parameter['value'] : '');
```

Comment:
PHP 7 and above only, so I believe we can just write

```suggestion
                                        return $key.'='.($parameter['value'] ?? '');

```

Reply:
I had taken this snippet from later down in the same file... Changed both now.